tests/integration/adapter/materialized_view/test_materialized_view.py

```diff
@@ -191,3 +220,78 @@ def test_update_full_refresh(self, project):
             f"select distinct hacker_alias from {schema}.hackers where name = 'Dade'", fetch="all"
         )
         assert len(result) == 2
+
+
+class TestMultipleMV:
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        """
+        we need a base table to pull from
+        """
+        return {
+            "people.csv": PEOPLE_SEED_CSV,
+            "schema.yml": SEED_SCHEMA_YML,
+        }
+
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "hackers.sql": MV_MODEL,
+        }
+
+    def test_create(self, project):
```

Comment:
Could you also test for updates and full refresh?

Reply:
Yeah I think if you change the model to add or rename MVs, then old ones will be left running, which is incorrect.